src/components/Model/CurrentModelName.js

```diff
@@ -23,8 +23,12 @@ class CurrentModelName extends Component {
             <Message color='olive'>
                     {t('dataset.common.currentDatasetLabel') + datasetName}
                     <br />
+                {pronDictName &&
+                    <>
```

Comment:
What does this empty tag do?

Reply:
This is shorthand for a "fragment", it wraps sibling expressions/components without adding extra DOM elements. 

"Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>"

The expression that is being wrapped here displays the prondict name and a line break. For example:
```
<p>name</p>
<br />
```

For this to be valid JSX, we require a parent element for the `p` and `br` sibling elements. We used to have to do this:
```
<div>
<p>name</p>
<br />
</div>
```

but that adds extra DOM unnecessarily and the amount of times it happens, the DOM gets littered with divs.

Fragments look like this
```
<React.Fragment>
<p>name</p>
<br />
</React.Fragment>
```

and shorthand version replaces `<React.Fragment>` with `<>`